srcpkgs/gnirehtet/files/gnirehtet/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
```

Comment:
Sourcing the config is useless, because the variables from it are not used anywhere in the service. Either remove this line or add `$OPTS` to the gnirehtet parameters (if that makes sense in this case).